tests/cases/rest_decorator_test.py

```diff
@@ -19,16 +19,20 @@
 
 import json
 import os
+import requests
```

Comment:
It's already a core requirement, so we don't strictly need to, but maybe for explicitness...? I'll let you decide.